gui/slick/views/snatchSelection.mako

```diff
@@ -458,7 +458,7 @@
                 % for hItem in provider_results['found_items']:
                     <%
                         # Hides special episodes from season search
-                        if manual_search_type == 'season' and 'E00' in hItem["name"]:
+                        if manual_search_type == 'season' and u'E00' in hItem["name"]:
```

Comment:
In this case it doesn't matter, but generally we should always be comparing unicode to unicode and bytes to bytes. I removed it now to keep consistency with the rest of the mako files.